src/vs/workbench/contrib/inlineChat/test/browser/inlineChatSession.test.ts

```diff
@@ -182,7 +186,8 @@ suite('InlineChatSession', function () {
 
 	test('Create, release', async function () {
 
-		const session = await inlineChatSessionService.createSession(editor, { editMode: EditMode.Live }, CancellationToken.None);
+		assert.ok(inlineChatService.getAllProvider().length > 0);
```

Comment:
can we move this logic into `setup` and have a variable `defaultProvider` that we use instead

Reply:
Yes and no, each test changes some of the provider functions to properly mock them out. Maybe we can have a function though that allows passing in the things we care about